docs/source/tutorials/image.ipynb

```diff
@@ -197,46 +221,290 @@
+    "            optimizer.step()\n",
+    "\n",
+    "            running_loss += loss.detach().cpu().item()\n",
+    "\n",
+    "        # Get accuracy on the test set\n",
+    "        accuracy = get_test_accuracy(model, testloader)\n",
+    "\n",
+    "        if accuracy > best_test_accuracy:\n",
+    "            best_epoch = epoch\n",
+    "\n",
+    "        # Condition for early stopping\n",
+    "        if epoch - best_epoch > patience:\n",
+    "            print(f\"Early stopping at epoch {epoch + 1}\")\n",
+    "            break\n",
+    "\n",
+    "        end_epoch = time.time()\n",
+    "\n",
+    "        print(\n",
+    "            f\"epoch: {epoch + 1} loss: {running_loss / len(trainloader):.3f} test acc: {accuracy:.3f} time_taken: {end_epoch - start_epoch:.3f}\"\n",
+    "        )\n",
+    "    return model\n",
+    "\n",
+    "\n",
+    "# Method for computing out-of-sample embeddings\n",
+    "def compute_embeddings(model, testloader):\n",
+    "    embeddings_list = []\n",
+    "\n",
+    "    with torch.no_grad():\n",
+    "        for data in tqdm(testloader):\n",
+    "            images, labels = data[0].to(device), data[1].to(device)\n",
+    "\n",
+    "            embeddings = model.embeddings(images)\n",
+    "            embeddings_list.append(embeddings.cpu())\n",
+    "\n",
+    "    return torch.vstack(embeddings_list)\n",
+    "\n",
+    "\n",
+    "# Method for computing out-of-sample predicted probabilities\n",
+    "def compute_pred_probs(model, testloader):\n",
+    "    pred_probs_list = []\n",
+    "\n",
+    "    with torch.no_grad():\n",
+    "        for data in tqdm(testloader):\n",
+    "            images, labels = data[0].to(device), data[1].to(device)\n",
+    "\n",
+    "            outputs = model(images)\n",
+    "            pred_probs_list.append(outputs.cpu())\n",
+    "\n",
+    "    return torch.vstack(pred_probs_list)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As some cleanlab features require scikit-learn compatibility, we adapt the above PyTorch neural net accordingly. [skorch](https://skorch.readthedocs.io) is a convenient package that helps with this. Alternatively, you can also easily wrap an arbitrary model to be scikit-learn compatible as demonstrated [here](https://github.com/cleanlab/cleanlab#use-cleanlab-with-any-model-for-most-ml-tasks)."
+    "## 4. Prepare the dataset for K-fold cross-validation \n",
+    "\n",
+    "To find label issues based on `pred_probs`, we recommend out-of-sample predictions, which can be produced via K-fold cross-validation. To ensure this tutorial runs quickly, we set K and other important neural network training hyperparameters to small values here. Use larger values to get good results in practice! \n",
+    "\n",
+    "Check [Computing Out-of-Sample Predicted Probabilities with Cross-Validation](https://docs.cleanlab.ai/stable/tutorials/pred_probs_cross_val.html) for a detailed tutorial."
```

Comment:
just move the link to be on the text: `via K-fold cross-validation`. No need to have the extra sentence: "Check ..."

Also note: this is not the right way to format internal links in tutorials, look at DEVELOPMENT.md and other tutorials, or ask @elisno